copy_and_paste: Change copy div css to match "day mode".

vrongmeal opened the pull request:
Color and background is made according to "day mode" exclusively here because when copying the content into, say, Gmail compose box, the styles come along.

This is done to avoid copying the content with dark background when using the app in night mode.

vrongmeal commented:
@timabbott This turned out to be pretty easy. Since all the other night mode message content styles were wrapped inside a `.message_content` selector, I simply just had to add the same parent to the syntax highlight part where it wasn't. Since the copy div we create does not have this class, changing just the foreground to black and background to white worked!

timabbott commented on static/styles/pygments.scss:
```diff
@@ -213,7 +213,7 @@
 
 /* Syntx Highlighting for night-mode */
-body.night-mode {
+body.night-mode .message_content {
```
Check whether this breaks the "message formatting" documentation -- we might need to add a `message_content` selector there.

timabbott commented:
This looks generally great.  The one thing I'm worried about is the codehilite change; for context, we have a few places we display Zulip markdown in-app and might need this styling:
* The in-app documentation for markdown formatting
* Markdown-rendered stream descriptions
* Drafts + message edit history, which I think are covered.

vrongmeal commented:
@timabbott I just updated this:

* The first two commits basically fix the style issue with current in-app docs (night-mode styles did not apply to inline code and code block)

* I added all the selectors which were used for inline `code` in `night_mode.scss` to `pygments.scss` for night mode syntax highlighting (This should now cover every place where syntax highlighting might be used).

timabbott commented:
@vrongmeal I merged the first 3 commits, thanks!  I think for the remainder, we should just make a single CSS class that we add to all of these places; see my comment in chat for details.

vrongmeal commented:
@timabbott I updated this with adding `rendered_markdown` class.

timabbott commented:
This is great, merged, thanks @vrongmeal!  I think there's probably a useful follow-up to look at the other CSS blocks we have for `message_content` and see if they can be migrated to `rendered_markdown` as well.

andersk commented:
Adding `rendered_markdown` to the entire `<div id="settings_page">` resulted in layout issues: the progressive tables no longer fill the width of the scrolling div that wraps them (#12435). Can we be more specific? Where in Settings do we actually render Markdown?

andersk commented:
(To be fair, that issue wasn’t visible until 27b51683952f8cf62a9b672e54c865d3a1545524. I still think we should be more specific though.)

timabbott commented:
Repling to discussion on #12435.
